snap/snapcraft.yaml

```diff
@@ -160,35 +160,44 @@ parts:
     plugin: autotools
     source: https://git.launchpad.net/ubuntu/+source/network-manager
     source-type: git
-    source-branch: applied/ubuntu/jammy-updates
+    source-branch: applied/ubuntu/noble
     build-packages:
-      - intltool
+      - gettext
       - libglib2.0-dev
       - ppp-dev
+      - libselinux1-dev
+      - libaudit-dev
       - libgnutls28-dev
       - uuid-dev
-      - systemd
+      - systemd-dev
       - libsystemd-dev
       - libudev-dev
       - libgirepository1.0-dev
       - gobject-introspection
       - python3-gi
       - libpsl-dev
       - libcurl4-gnutls-dev
+      - gtk-doc-tools
       - libglib2.0-doc
       - libmm-glib-dev
       - libndp-dev
       - libreadline-dev
       - libnewt-dev
       - libteam-dev
       - libjansson-dev
-      - libnss3-dev
+      - libbluetooth-dev
+      - valac
+      - polkitd
+      - libpolkit-gobject-1-dev
```

Comment:
Is this really needed? We are disabling polkit in the build.

Reply:
Yep, they are still required for the build, without them we get

`/usr/bin/msgfmt: cannot locate ITS rules for data/org.freedesktop.NetworkManager.policy.in`